android/modules/ui/src/java/ti/modules/titanium/ui/android/DrawerLayoutProxy.java

```diff
@@ -30,6 +28,12 @@
 	public static final int LOCK_MODE_UNLOCKED = DrawerLayout.LOCK_MODE_UNLOCKED;
 	@Kroll.constant
 	public static final int LOCK_MODE_UNDEFINED = DrawerLayout.LOCK_MODE_UNDEFINED;
+	@Kroll.constant
+	public static final int GRAVITY_BOTH = 1;
```

Comment:
Shouldn't this be equivalent to Gravity.CENTER_HORIZONTAL? Maybe the name and value should better mirror the underlying Gravity constant here?